test/browser/webgl2_simple_enable_extensions.c

```diff
@@ -61,6 +61,15 @@ int main()
   if (hasext(exts, "WEBGL_multi_draw_instanced_base_vertex_base_instance"))
     assert(emscripten_webgl_enable_WEBGL_multi_draw_instanced_base_vertex_base_instance(context));
 
+  if (hasext(exts, "EXT_polygon_offset_clamp"))
+    assert(emscripten_webgl_enable_EXT_polygon_offset_clamp(context));
+
+  if (hasext(exts, "EXT_clip_control"))
+    assert(emscripten_webgl_enable_EXT_clip_control(context));
+
+  if (hasext(exts, "WEBGL_polygon_mode"))
+    assert(emscripten_webgl_enable_WEBGL_polygon_mode(context));
```

Comment:
It lots of places in this PR I see only 3 extensions (looks like `EXT_depth_clamp` is missing here and elsewhere).  Is there some reason for that?